services/r2ctl/main/main.go

```diff
@@ -90,12 +89,10 @@ func main() {
 		SetMetricsSamplingRate(cfg.Metrics.SampleRate()).
 		SetReportInterval(cfg.Metrics.ReportInterval())
 
-	clockOpts := clock.NewOptions()
-
 	listenAddr := fmt.Sprintf("%s:%d", cfg.HTTP.Host, cfg.HTTP.Port)
 	serverOpts := cfg.HTTP.NewHTTPServerOptions(instrumentOpts)
 
-	store, err := cfg.Store.NewR2Store(instrumentOpts, clockOpts)
```

Comment:
what happened to the clockOpts?

Reply:
It's initialized to `clock.NewOptions()` by default and it's primarily useful to testing. I don't imagine we'll need to configure it in the config, and if we do, we can always add them back.